content/services/development.md

```diff
@@ -9,9 +9,180 @@ action:
   text: See work samples »
   url: /work/
 summary: |
-  **We integrate design and development in an agile process**,
-  so you only need to hire one team --
-  completely focused on achieving your goals
-  with smooth and efficient progress
-  from concept to launch and beyond.
+  **We integrate design and development in an agile process**, so you only need to hire one team – completely focused on achieving your goals with smooth and efficient progress from concept to launch and beyond. Starting from user needs, we collaborate with you to deliver usable features quickly and efficiently.
+
+  We provide expertise across a range of technologies, including **Python/Django**, accessible **HTML**, modern **Sass/CSS**, **Vue**, **Node**, **Backbone/Marionette**, and more.
 ---
+
+{% import 'embed.macros.njk' as embed %}
+{% import 'quotes.macros.njk' as quotes %}
+{% import 'layout.macros.njk' as layout %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['realistic', 'curious']
+) }}
+
+{{ layout.title('Services') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    content=embed.img(
+      src='pages/trig-phone.jpg',
+      alt='trig race screen',
+      width=744 ,
+      height=1611,
+      sizes='media'
+    )
+  )
+) %}
+
+{% call layout.grid('narrow-columns') %}
+
+{% call layout.block('item') %}
```

Comment:
maybe call this `grid-item` or `column-item` or something that connects it to the `narrow-columns` grid? Alternately, consider using yaml data for these, like we do with icon blocks? We don't need to go that direction, but I tend to consider it when we otherwise need nested `call`s…

Reply:
I had it in the yaml, but I think it is more readable when the content is the order it appears on the page in general (plus I tend do always mess something up when it comes to yaml). I know we have content in the front matter for many parts of the site. I am old school ¯\_(ツ)_/¯

Reply:
lol, yeah - I agree it's a trade-off. I'm just careful about nested calls, because I never understand how they handle context. But this seems to work, so I guess you got it right! 😂

Reply:
(Sidenote: Nunjucks is buggy and I don't trust it 😬)

Reply:
@mirisuzanne A few of the bugs from v3.0.0 have been fixed by v3.2.0... but there are still a number of bugs with scoping, especially as it relates to `call` and `set` blocks:

- https://github.com/mozilla/nunjucks/issues/679
- https://github.com/mozilla/nunjucks/issues/912
- https://github.com/mozilla/nunjucks/issues/664
- https://github.com/mozilla/nunjucks/issues/1131

Reply:
@jgerigmeyer are you asking to change this to using the yaml content instead of what is in here now? I'm not sure what you are saying.

Reply:
@stacyk I wasn't saying that specifically, no. But I think we should all be vaguely aware of the existing bugs in Nunjucks, and hand-test any uses of `caller` (especially nested usage) or `set` to be sure it's actually working as we want.